spec/lmi_for_all_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+require 'lmi_for_all'
+
+describe LmiForAll do
+  let(:lmi_client) do
+    double(
+      :lmi_client,
+      soc_code_lookup: soc_code_details.deep_stringify_keys,
+      hours_lookup: hours_details,
+      pay_lookup: pay_details
+    )
+  end
+  let(:soc_code_details) do
+    {
+      title: title,
+      description: description,
+      add_titles: additional_titles,
+      tasks: tasks,
+      qualifications: qualifications
+    }
+  end
+  let(:hours_details) { double(:hours_details) }
+  let(:pay_details) { double(:pay_details) }
+  let(:title) { 'Occupation title' }
+  let(:description) { 'Occupation description' }
+  let(:tasks) { 'Occupation tasks' }
+  let(:qualifications) { 'Occupation qualifications' }
+  let(:additional_titles) do
+    [title_too_short] + [title_with_parenthesis] + [valid_title] * 30
+  end
+  let(:title_too_short) { 'no' }
+  let(:title_with_parenthesis) { 'Occupation (with parenthesis)' }
+  let(:valid_title) { 'occupation, with, commas, manager' }
+  let(:soc_code) { double(:soc_code) }
+  let(:result) { described_class.new(lmi_client).lookup(soc_code) }
```

Comment:
can we make this a named subject (`subject(:result)`) to make it clear that this is what the tests are about